src/FSharp.Core/printf.fs

```diff
@@ -2,6 +2,8 @@
 
 namespace Microsoft.FSharp.Core
 
+#nowarn "1118"  // 'Make' marked 'inline', perhaps because a recursive value was marked 'inline'
```

Comment:
Because an Api that used to inline a private value (which becomes internal when fsharp.core is compiled realsig-) now there is a warning instead when compiled realsig+.  So this just turns off the warning.  The warning tells us that the value is no longer inlined, because it relies on a private value.  The alternate fix would be to make the value internal rather than public, i chose to disable the warning.

Reply:
Can't we then make the value internal instead?
I do not know how many APIs that are, but if it is a scenario which relies on type specialization (e.g. inlining of type equality), this could have big performance implications.

I don't think we should silently put in potential performance degradations.
(e.g. List.contains used to have a missed equality inlining, the perf difference was something like 16-fold for a list of integers.

Reply:
Sure we could.